docs/demo.md

```diff
@@ -0,0 +1,195 @@
+
+### Setup Kai VSCode IDE plugin
+
+- Follow along the steps listed in [here](https://github.com/konveyor-ecosystem/kai/tree/main/ide) to intergrate Kai IDE plugin with VSCode.
+
+- Before starting the Kai server, Select model `meta-llama/llama-3-70b-instruct` by adding this to `kai/config.toml` file
+
+```yaml
+provider = "IBMOpenSource"
+args = { model_id = "meta-llama/llama-3-70b-instruct", max_new_tokens = 2048 }
+```
+
+If you are interested in using `gpt-3.5-turbo` refer to the section below,
+
+#### Running Kai with GPT-3.5
+
+- Export your OpenAI api key `export OPENAI_API_KEY="<your-openai-api-key>"`
+
+- Before starting the Kai server, Select model `gpt-3.5-turbo` by adding this to `kai/config.toml` file
+
+```yaml
+provider = "OpenAI"
+args = { model_id = "gpt-3.5-turbo" }
+```
+
+- Run the server using `make run-server`
+
+## Step 2: Demo
+
+As mentioned earlier, in this demo we will show various capabilities of Kai. We will begin with the example - Change import namespaces. Kai uses hints from the anlaysis results and replaces the javax persistence with Jakarta persistence libraries in the following files:
+
+- `src/main/java/com/redhat/coolstore/model/ShoppingCart.java`
+- `src/main/java/com/redhat/coolstore/model/InventoryEntity.java`
+
+Moving on to slightly more involved tasks, Kai will be migrating the file to use the correct Quarkus cdi bean.
+
+- `src/main/java/com/redhat/coolstore/service/CatalogService.java`
+
+Lastly, for the example - EJB Remote and Message Driven Bean(MBD) changes , we will use the following two files, and we will dive deeper into it later in the demo
+
+- `src/main/java/com/redhat/coolstore/service/ShippingService.java`
+- `src/main/java/com/redhat/coolstore/service/ShoppingCartOrderProcessor.java`
+
+Through these examples, we aim to provide insights into Kai's ability to address varying degrees of migration challenges within application codebases.
+
+### 2.1 Clone the coolstore app
+
+In this step, we will clone the Coolstore application, which we will be used demo the migration process to Quarkus.
+
+1. To clone the Coolstore demo from its repository -
+
+   ```bash
+   git clone https://github.com/konveyor-ecosystem/coolstore.git
+   ```
+
+2. Next, switch to the branch of the Coolstore app that's been partially migrated -
+
+   ```bash
+   git checkout partial-migration
+   ```
+
+### 2.2 Run Analysis
+
+We will kick off by analyzing the Coolstore application using the following targets: cloud-readiness, Jakarta EE, and Quarkus.
+
+1. Open VSCode and load coolstore project if it is not already loaded.
+2. Follow steps from 1 through 4 from https://github.com/konveyor-ecosystem/kai/tree/main/ide#usage-summary
+3. Once the analysis is complete you will see incidents listed on the following files
+
+- `src/main/java/com/redhat/coolstore/model/ShoppingCart.java`
+- `src/main/java/com/redhat/coolstore/model/InventoryEntity.java`
```

Comment:
I don't see InventoryEntity.java, I see 'Order.java'.

Reply:
merged the branch and it should show up now. When you get sometime can you check? Thank you